Lecture_14/Homework_14.py

```diff
@@ -0,0 +1,48 @@
+import re
+
+phone_book = {
+    'Taras': '+380123456789',
+    'Dmutro': '+380123456790',
+    'Kevin': '+380123456791',
+    'Peter': '+380123456792',
+}
+while True:
+    input_data = input('Please, enter new command: ')  # format "command name number" commands: stats, add, delete,
+    # list, show
+    split_input: list[str] = input_data.split()
+
+    command = split_input[0]
+    if len(split_input) > 1:
+        name = split_input[1]
+    if len(split_input) > 2:
+        number = split_input[2]
+
+        match1 = re.findall(r"\+380\d{9}(?=[, ]|$)", number)
```

Comment:
can be something like that
```suggestion
        match = re.findall(r"(?:\+?38|8)?0\d{9}(?=[, ]|$)", number)
```

Reply:
test your patterns here [regex101.com](https://regex101.com/)
it is a very convenient tool to test

Reply:
Дуже дякую, буду звертатись до цього сайту!